lib/log.js

```diff
@@ -1,9 +1,13 @@
+// Native
+const { join } = require('path')
+
 // Packages
 const chalk = require('chalk')
 const jsome = require('jsome')
 const PrettyError = require('pretty-error')
 const stringLength = require('string-length')
-const { json, send } = require('micro')
+
+const { json, send } = require(join(process.cwd(), 'node_modules/micro'))
```

Comment:
You can just require micro. There’s no need to change this. Same Goes for the other cases. Also add micro to the devdeps 👍🏻